src/utils.js

```diff
@@ -441,6 +443,98 @@ function safeJoin(input, delimiter) {
   return output.join(delimiter);
 }
 
+// Default Node.js REPL depth
+var MAX_SERIALIZE_EXCEPTION_DEPTH = 3;
+// 50kB, as 100kB is max payload size, so half sounds reasonable
+var MAX_SERIALIZE_EXCEPTION_SIZE = 50 * 1024;
+var MAX_SERIALIZE_KEYS_LENGTH = 40;
+
+function utf8Length(value) {
+  return ~-encodeURI(value).split(/%..|./).length;
```

Comment:
are you only using `~-` this for subtracting 1? I trust it's right, just want to understand.

Reply:
Correct :) https://www.joezimjs.com/javascript/great-mystery-of-the-tilde/

Reply:
why not use `).length - 1`? (it's NBD, just curious)

Reply:
Because I trust in every line of code that Substack writes tbh, nothing more - https://github.com/substack/utf8-length/blob/master/index.js :P